pkgs/l4t/l4t-opencv.nix

```diff
@@ -0,0 +1,41 @@
+{ buildFromDebs
```

Comment:
This is future work, but we should really strive to build opencv from source if at all possible.